npm/ng-packs/packages/identity/src/lib/components/users/users.component.html

```diff
@@ -77,119 +77,118 @@ <h5 class="card-title">{{ 'AbpIdentity::Users' | abpLocalization }}</h5>
   </div>
 </div>
 
-<ng-template #modalContent let-modal>
-  <div class="modal-header">
-    <h5 class="modal-title" id="modal-basic-title">
-      {{ (selected?.id ? 'AbpIdentity::Edit' : 'AbpIdentity::NewUser') | abpLocalization }}
-    </h5>
-    <button type="button" class="close" aria-label="Close" (click)="modal.dismiss()">
-      <span aria-hidden="true">&times;</span>
-    </button>
-  </div>
-  <form [formGroup]="form" (ngSubmit)="save()">
-    <div class="modal-body">
-      <ngb-tabset>
-        <ngb-tab [title]="'AbpIdentity::UserInformations' | abpLocalization">
-          <ng-template ngbTabContent>
-            <div class="mt-2">
-              <div class="form-group">
-                <label for="user-name">{{ 'AbpIdentity::UserName' | abpLocalization }}</label
-                ><span> * </span>
-                <input type="text" id="user-name" class="form-control" formControlName="userName" />
-              </div>
+<abp-modal [(visible)]="isModalVisible" *ngIf="isModalVisible" [centered]="true">
```

Comment:
Centered input default value is `true`. You can remove `[centered]="true"`

We add structural directives near to HTML tag. Replace below
```html
<abp-modal [(visible)]="isModalVisible" *ngIf="isModalVisible"
```